packages/esm-outpatient-app/src/index.ts

```diff
@@ -67,6 +67,13 @@ function setupOpenMRS() {
         online: true,
         offline: true,
       },
+      {
+        id: 'edit-queue-entry-status-modal',
+        load: getAsyncLifecycle(() => import('./active-visits/change-status-dialog.component'), {
+          featureName: 'end visit',
+          moduleName,
+        }),
```

Comment:
Not sure why the featureName is called `end visit`, maybe you can provide some clarifications